apps/prairielearn/src/pages/instructorAssessmentManualGrading/instanceQuestion/instanceQuestion.ejs

```diff
@@ -12,7 +12,7 @@
     <script src="<%= asset_path('javascripts/lodash.min.js') %>"></script>
     <script src="<%= asset_path('javascripts/require.js') %>"></script>
     <script src="<%= asset_path('localscripts/question.js') %>"></script>
-    <script src="<%= asset_path('localscripts/question${effectiveQuestionType}.js') %>"></script>
+    <script src="<%= asset_path(`localscripts/question${effectiveQuestionType}.js`) %>"></script>
```

Comment:
No worries, it wasn't working at all anyway, because of the lack of file/text paths. Legacy questions tend not to get that much attention in testing new features, I only tested it because I was testing a v2 question with MathJax in #7926, and decided to test it in the manual grading interface.